### PR TITLE
docs: add reproducible example

### PR DESCRIPTION
**Pull request by jnodorp-jaconi**
Keycloak only demo.

### CODE REVIEW

File: example/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+version: "3.9"
+services:
+  morp:
+    image: ghcr.io/jaconi-io/morp:0.2.3
```
Comment (jfiedler-jaconi) [inline example/docker-compose.yaml:4]: Should we reference `latest` here to make sure the latest image is being pulled?

File: README.md [hunk 2]
```diff
@@ -45,25 +45,22 @@ MORP "glue".
...
+
+This project contains an example showing how to protect a httpbin instance using authentication with Keycloak. Run the
+example by switching to the example folder and using `docker compose`:
+
+```shell
+cd example
+docker compose up
 ```
 
+Open [http://foo.localtest.me:8080](http://foo.localtest.me:8080) in your browser and log in with the username
```
Comment (jfiedler-jaconi) [inline README.md:58]: Does the localtest.me domain require any local setup to work (like /etc/hosts tricks)? If so we should document it.

**Review by jfiedler-jaconi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jnodorp-jaconi) [reply README.md:58]: No. `*.localtest.me` just resolves to 127.0.0.1.

Comment (jnodorp-jaconi) [reply README.md:58]: See http://readme.localtest.me

File: example/docker-compose.yaml (hunk 1 above)
Comment (jnodorp-jaconi) [reply example/docker-compose.yaml:4]: We don't publish a latest image (yet). This gets updated automatically, though. See my addition to `.releaserc`.

**Pull request merged**